### PR TITLE
Fixes the area around medbay on boxstation

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 Pretty much what the title says

## Why It's Good For The Game
 Makes medbay neater and makes it the areas sound better then they used to be

## Images of changes
 
Screenshot 2024-01-31 203144

## Testing
  Spawned as the CMO and checked if the APC's were named right and they were on Box station

## Changelog
:cl:
Tweak: Cyberiad Medbay's, medbay secondary treatment are now unique areas named patients rooms secondary and patients room
Tweak: Fixed the area at medbay on box station
/:cl:

### CODE REVIEW

**Comment by Fluffyzack21** [comment]
the floating APC you see on map diff is wrong on the floor, I tripled check then loaded paradise on the test server and it was not floating for some reason map diff did not update and read the new APC spot
Screenshot 2024-01-31 203605

File: code/game/area/ss13_areas/medical_areas.dm [hunk 1]
```diff
@@ -39,6 +39,16 @@
 	icon_state = "patients"
 	sound_environment = SOUND_AREA_SMALL_SOFTFLOOR
 
+/area/station/medical/patients_rooms1
+	name = "\improper Patient's Rooms"
```
Comment (CinnamonSnowball) [inline code/game/area/ss13_areas/medical_areas.dm:43]:
```suggestion
	name = "\improper Patient Room"
```

**Comment by CinnamonSnowball** [comment]
Also, the changelog is a bit vague, you should probably mention what the new area *is* and which rooms you're fixing
Players don't really need to know how the PR affects VSC, especially when it doesn't really

**Comment by Fluffyzack21** [comment]
> Also, the changelog is a bit vague, you should probably mention what the new area _is_ and which rooms you're fixing Players don't really need to know how the PR affects VSC, especially when it doesn't really

fixed

**Review by Henri215 — CHANGES_REQUESTED** [review]
image

There is already one patient room area that is not being used on Cyberiad, you would only have to create one more, not two.


image


This is not being used on cyberiad either, do you think it could be used or no?

**Review by S34NW — CHANGES_REQUESTED** [review]
image
image

Don't stack wall furniture

**Comment by Fluffyzack21** [comment]
> image
> 
> There is already one patient room area that is not being used on Cyberiad, you would only have to create one more, not two.
> 
> image
> 
> This is not being used on cyberiad either, do you think it could be used or no?

I would have to name them one and 2 due to having two of the same rooms

**Review by CinnamonSnowball — CHANGES_REQUESTED** [review]
The changelog is still kinda vague; you don't state *which* area is being fixed/tweaked and how, and the `Add:` line doesn't really mean anything to a regular player.

You could do something like "Tweak: Cyberiad Medbay's `<affected rooms>` are now unique areas named `<new area names>`"

File: code/game/area/ss13_areas/medical_areas.dm [hunk 2]
```diff
@@ -39,6 +39,16 @@
 	icon_state = "patients"
 	sound_environment = SOUND_AREA_SMALL_SOFTFLOOR
 
+/area/station/medical/patients_rooms1
+	name = "\improper Patient Room"
+	icon_state = "patients"
+	sound_environment = SOUND_AREA_SMALL_SOFTFLOOR
+
+/area/station/medical/patients_rooms2
+	name = "\improper Patient's Rooms"
```
Comment (CinnamonSnowball) [inline code/game/area/ss13_areas/medical_areas.dm:48]: You forgot to tweak the other area's name as well

Comment (Fluffyzack21) [reply code/game/area/ss13_areas/medical_areas.dm:47]: fixed

**Comment by Fluffyzack21** [comment]
> image image
> 
> Don't stack wall furniture

fixed

**Review by S34NW — APPROVED** [review]

**Pull request merged**